### PR TITLE
Implemented Pressure Points Notable

### LINKED ISSUE #367: Implement Pressure Points Notable
First mod is not parsed

```
Your Critical Strikes have a 5% chance to deal Double Damage
40% increased Critical Strike Chance
```

### PR DESCRIPTION
**Pull request by Nostrademous**
Fixes #367 

Implemented as new mod since we can't check DoubleDamageChance of a Crit direclty. It doesn't do anything... I tried.

### CODE REVIEW

File: Modules/CalcOffence-3_0.lua [hunk 1]
```diff
@@ -1016,7 +1016,8 @@ function calcs.offence(env, actor, activeSkill)
 
 		-- Calculate Double Damage + Ruthless Blow chance/multipliers
 		output.DoubleDamageChance = m_min(skillModList:Sum("BASE", cfg, "DoubleDamageChance") + (env.mode_effective and enemyDB:Sum("BASE", cfg, "SelfDoubleDamageChance") or 0), 100)
-		output.DoubleDamageEffect = 1 + output.DoubleDamageChance / 100
+		output.CritDoubleDamageChance = m_min(skillModList:Sum("BASE", cfg, "CritDoubleDamageChance"), 100)
+		output.DoubleDamageEffect = 1 + (output.DoubleDamageChance + (output.CritDoubleDamageChance*output.CritChance/100)) / 100
```
Comment (NickRyder) [inline Modules/CalcOffence-3_0.lua]: IIRC there are separate code paths for crit and non crits in CalcOffesnse, which are important due to a bunch of other conditionals which depend on crits.

Comment (Nostrademous) [reply Modules/CalcOffence-3_0.lua]: are you saying this is wrong or that you need more verification that this change is correct?

Comment (NickRyder) [reply Modules/CalcOffence-3_0.lua]: I believe this is wrong - although I still might be missing something.
In the scenario where you deal `non_crit_damage` and `crit_damage` the total expected dps should be `crit_chance * (crit_double_damage_multi) * crit_damage + (1- crit_chance) * (non_crit_double_damage_multi) * non_crit_damage`

I believe the way you have it written is `[crit_chance * (crit_double_damage_multi) + (1- crit_chance) * (non_crit_double_damage_multi)]* [crit_chance  * crit_damage + (1- crit_chance) * non_crit_damage]`

**Comment by Nostrademous** [comment]
@NickRyder try this PoB: [https://pastebin.com/Lh9R1nd1](https://pastebin.com/Lh9R1nd1)

In that example my Crit chance is 5%
Setup

If you look at the Calcs tab, you will see that the chance to deal double damage is scaled by the crit chance and only reported as 2.5% (displayed as a multiplier of 1.025 ... displayed as a 1.02 since we round our string) to the damage.
CritComponent

If you play around with changing the Jewel to other values and even adding crit chance it still, as least to me and how I understand it to work, displays the correct impact. Like if you set chance to deal double damage to 100% your multiplier is 1.05 since you have a 5% crit chance.

**Comment by Nostrademous** [comment]
Did find one bug where I didn't clamp the max DD effect at a 2.0 multiplier. Fixed.

**Comment by Nostrademous** [comment]
Closing as fixed in #382

**Pull request closed without merging**